### ISSUE TITLE
Can not make a call if a stream is given at first.

### ISSUE THREAD

**Issue by zeng450026937** [versatica/JsSIP#509]
If A stream is given, the promise queue can't be resolve correctly in _sendInitialRequest()

**Comment by ibc** [comment]
Can you provide MUCH more info, please?

**Comment by zeng450026937** [comment]
RTCSession._sendInitialRequest()
```js
    Promise.resolve()
      // Get a stream if required.
      .then(() =>
      {
        // A stream is given, let the app set events such as 'peerconnection' and 'connecting'.
        if (mediaStream)
        {
          setTimeout(() => Promise.resolve);
        }
        // Request for user media access.
        else if (mediaConstraints.audio || mediaConstraints.video)
        {
          this._localMediaStreamLocallyGenerated = true;

          return navigator.mediaDevices.getUserMedia(mediaConstraints)
            .catch((error) =>
            {
              if (this._status === C.STATUS_TERMINATED)
              {
                throw new Error('terminated');
              }

              this._failed('local', null, JsSIP_C.causes.USER_DENIED_MEDIA_ACCESS);

              debugerror('emit "getusermediafailed" [error:%o]', error);

              this.emit('getusermediafailed');

              throw new Error('getUserMedia() failed');
            });
        }
      })
      .then((stream) =>
      {
           ....
      }
```
```js
 if (mediaStream)
 {
    // resolve nothing
    setTimeout(() => Promise.resolve);
 }
```
No media stream will be added to the peerconnection.
In my case, the offerConstraints for  connection.createOffer(), offerToReceiveAudio and offerToReceiveVideo are both false.
The promise queue will never resolve or reject.(maybe blocked in createOffer)

**Comment by ibc** [comment]
Thanks. It should be fixed in v3.2.7.
